tagstudio/src/qt/modals/drop_import.py

```diff
@@ -0,0 +1,79 @@
+import os
+import shutil
+import typing
+
+from PySide6.QtGui import QDropEvent, QDragEnterEvent, QDragMoveEvent
+from PySide6.QtWidgets import QMessageBox
+
+if typing.TYPE_CHECKING:
+    from src.qt.ts_qt import QtDriver
+
+
+def dropEvent(driver: "QtDriver", event: QDropEvent):
+    if event.mimeData().hasUrls():
+        if event.mimeData().urls()[0].isLocalFile():
+            urls = event.mimeData().urls()
+            duplicate_filesnames = []
+            for url in urls:
+                if os.path.exists(driver.lib.library_dir + "/" + url.fileName()):
```

Comment:
there's another PR with replacing `os.path.` with `pathlib.Path`. Would be nice to have it everywhere for consistency